nixos/modules/services/misc/synergy.nix

```diff
@@ -46,17 +48,20 @@ in
         enable = mkEnableOption "the Synergy server (send keyboard and mouse events)";
 
         configFile = mkOption {
+          type = types.path;
```

Comment:
I am not sure about this. Are files paths?

Reply:
Yes, paths are unmergable strings starting with '/'